pypsa/plot.py

```diff
@@ -33,6 +36,13 @@
 except ImportError:
     pltly_present = False
 
+explore_deps_present = True
+try:
+    import folium
+    import mapclassify
+except ImportError:
+    explore_deps_present = False
+
```

Comment:
Since we only use those dependencies for n.explore, I think we can move the import in the function

Reply:
good point, done!